relay-metrics/src/aggregation.rs

```diff
@@ -1140,6 +1158,11 @@ impl Aggregator {
             );
             total_bucket_count += bucket_count;
 
+            let mut sizes: Vec<usize> = Vec::new();
+            project_buckets
+                .iter()
+                .for_each(|bucket| sizes.push(bucket.value.relative_size()));
```

Comment:
_nit_: We usually write this as plain `for` loop, which seems more idiomatic:

```suggestion
            for bucket in &project_buckets {
                sizes.push(bucket.value.relative_size());
            }
```

Same below when logging the metrics.